src/components/forms/Combobox.tsx

```diff
@@ -818,7 +804,16 @@ export function ComboboxAsync<T = any>({
 
   useEffect(() => {
     if (data) {
-      setEntries([...data]);
+      setEntries(
```

Comment:
@beganovich The data was adjusted using `entryOptions` only if we had a case triggering the API. The `than` block will never be executed if we retrieve the data from cache memory. This is the reason for the change and the core of the problem why the label was not populated with the initial number property. So, the data needs to be just the response. However, if we need some adjustments, we can do that during the initial mounting of the combobox, even if the data is coming from the cache. Let me know your thoughts.